docs/embedded.md

```diff
@@ -0,0 +1,87 @@
+
+For an example, you can look at the [sample](https://github.com/microsoft/CsWinRT/tree/master/src/Samples/TestEmbedded). 
+
+## Scenarios
+
+This feature allows C# apps and libraries to target `net5.0` (and above), .NET Framework 4.6.1+,`netcoreapp3.1`, and `netstandard2.0` while also using the Windows SDK.
+Moreover, a library can target `netstandard2.0` and be able to run on NetFX, Net Core and Net 5. 
+
+## Usage 
+
+Using embedded support allows you to target non-Windows TFMs and older versions of .NET. 
+You will need to set a few properties:
+  * `CsWinRTEmbedded` - must be set to `true` to include the sources
+  * `CsWinRTWindowsMetadata` - must be set to the Windows SDK release to use for Windows APIs
+  * `LangVersion` - must be set to `9` because the WinRT.Runtime sources use C# 9
+
+Targeting `netstandard2.0` requires adding three package references, in order to use APIs that have been deprecated. 
+
+```xml
+<PackageReference Include="System.Memory" Version="4.5.4" />
+<PackageReference Include="System.Runtime.Extensions" Version="4.3.1" />
+<PackageReference Include="System.Numerics.Vectors" Version="4.4.0" />
+```
+
+Here is an example project file for a library cross-targeting and embedding WinRT support.
+
+```csproj
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net6.0;net5.0;netcoreapp3.1;netstandard2.0</TargetFrameworks>
+    <Platforms>x64;x86</Platforms>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <LangVersion>9</LangVersion>
+    <CsWinRTEmbedded>true</CsWinRTEmbedded>
+    <CsWinRTWindowsMetadata>10.0.19041.0</CsWinRTWindowsMetadata>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- either use the private produced by local build, or find a way to simulate cswinrt nuget reference -->
+    <PackageReference Include="Microsoft.Windows.CsWinRT" Version="1.3.6-prerelease.211004.13" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\path\CppwinrtComponent\CppwinrtComponent.vcxproj" />
+  </ItemGroup> 
+  
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="System.Memory" Version="4.5.4" />
+    <PackageReference Include="System.Runtime.Extensions" Version="4.3.1" />
+    <PackageReference Include="System.Numerics.Vectors" Version="4.4.0" />
+  </ItemGroup>
+    
+  <PropertyGroup>
+    <CsWinRTIncludes>
+      CppwinrtComponent;
+      Windows.Devices.Geolocation;
+      Windows.Foundation;
+    </CsWinRTIncludes>
+  </PropertyGroup>
+ 
+</Project>
+```
+
+There is a known issue consuming C++/WinRT authored WinRT Components in a library using embedded support 
+when the component is consumed via `ProjectReference`. 
+You will need to edit the .vcxproj file of all the components by enabling Windows Desktop Compatible. 

```

Comment:
Not a known issue but more of a requirement. Also instead of needing to directly edit the vcxproj, we can tell folks to check that the VS properties are set: In the project properties for the C++/WinRT component, under Configuration Properties > General > Project Defaults, set the property Windows Desktop Compatible to Yes for all configs/platforms